content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Gateway
+content_type: concept
+description: >-
+  Make network services available by using an extensible, role-oriented, protocol-aware
+  API. Gateway is a group of resources that provide dynamic infrastructure provisioning
+  and advanced traffic routing for Kubernetes services.
+weight: 55
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.29" state="stable" >}}
+
+Make network services available by using an extensible, role-oriented, protocol-aware configuration
+mechanism. The [Gateway][gw-api] API consists of [resources][resources] that provide dynamic
+infrastructure provisioning and advanced traffic routing.
+
+The Gateway project is maintained by the [SIG-NETWORK][sig-net] community outside of Kubernetes.
```

Comment:
```suggestion
The term _Gateway API_ refers to a family of extension APIs that you can deploy into your
Kubernetes cluster to gain the extra functionality.
```

Reply:
I removed this paragraph since Gateway API is defined in the glossary and again in the first paragraph of this page.